packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -75,7 +75,7 @@ function GridItem( { categoryId, item, ...props } ) {
 		postType: item.type,
 		postId: isUserPattern ? item.id : item.name,
 		categoryId,
-		categoryType: item.type,
+		categoryType: isTemplatePart ? item.type : PATTERN_TYPES.theme,
```

Comment:
Nit: This works but in my mind `isUserPattern ? PATTERN_TYPES.theme : item.type` should be closer to the logic here. This is just in case we have another pattern type that we want to add here in the future. Probably will never happen though 😅 .

Reply:
Fair point.

My thinking was that we'd be going the other way, consolidating all pattern types rather than adding more. At that stage, the category type wouldn't really be needed any more. In the interim, more types would be treated as per the `pattern` type.

Reply:
After a little discussion, we'll go with the current logic.